src/index.html

```diff
@@ -10,13 +10,110 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Dela+Gothic+One&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      href="images/favicon.ico"
+      type="image/x-icon"
+    />
+    <title>Moyo Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="container">
+        <ul>
+          <li>
+            <a
+              href="#appl"
+              class="box apple is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#sams"
+              class="box samsung is-active"
```

Comment:
There are multiple elements with the `is-active` class. If this class is intended to indicate a single active item, ensure only one element has this class to accurately represent the active state.